README.md

```diff
@@ -49,7 +49,9 @@ As an alternative solution, you can deploy mxnet-operator bypass ksonnet
 
 ```
 kubectl create -f manifests/crd-v1beta1.yaml 
```

Comment:
This will be migrated to `kustomize` to a follow PR. We still have some references to `ks` in the doc. We will clean them up together